docs/getting-started/quickstart.rst

```diff
@@ -31,7 +31,7 @@ You can download the Gammapy tutorial notebooks and the example
 datasets. The total size to download is ~180 MB. Select the location where you
 want to install the datasets and proceed with the following commands:
 
-.. code:: bash
+.. substitution-code-block:: console
```

Comment:
why is this change required? This is still giving the correct expected results no?

Reply:
If you look at the [dev version](https://docs.gammapy.org/dev/getting-started/index.html#recommended-setup), the release is showing as `|release|`

Reply:
Indeed